### DISCUSSION TITLE
Newest CRA 17+ with pinned error - Module not found: Can't resolve...

### DISCUSSION THREAD

**Discussion by Wozniaxos** [nolimits4web/swiper#5119] [Q&A]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

...

### Bug description

Hey, i see that conversation on that issue is blocked, do you plan to add working out of the box with newest CRA? it is weird that package which have react components doesn't work out of the box with newest React and most popular react app setup.

### Swiper version

7+

### Platform/Target and Browser Versions

macOs

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by inear** [comment]
I also try to find information how to actually use version 7 of this awesome library. I can't change the build-process in my project, which is created with vue-cli, and don't know the implications of running ESM only with webpack, babel and all that stuff that many webdevs like me let other boilerplate-frameworks take care of

**Comment by PauloJorgeS** [comment]
I'm getting the same problem I think.

I update react-scripts to "5.0.1", and this stop to work:
```js
import { Swiper } from "swiper/react/swiper-react.js";
import "swiper/swiper.min.css";
```

now I have to use like this:

```js
import { Swiper } from "swiper/react";
import "swiper/css";
```

And now is broken on my storybook build :/

It was cool that we could use the imports like the example in: [Swipper with create react app](https://swiperjs.com/react#usage-with-create-react-app),
